vendor/teleopti/calendar-data-updater.directive.js

```diff
@@ -0,0 +1,159 @@
+(function() {
```

Comment:
the vendor folder is more for the external libraries. Maybe it would be better to add those directives in a new js folder at the same level as css...etc